sdk/schema/plugin.go

```diff
@@ -78,12 +78,6 @@ func (p Plugin) Validate() (bool, ValidationReport) {
 		Severity:    ValidationSeverityError,
 	})
 
-	report.AddCheck(ValidationCheck{
-		Description: "Has no more than one executable defined. Plugins with multiple executables are not supported yet",
-		Assertion:   len(p.Executables) == 1,
-		Severity:    ValidationSeverityError,
-	})
```

Comment:
Another concern that I have about removing this check is that now, we may end up with a situation where a plugin contributor uses the latest version of `shell-plugin`'s main to develop a multi-exec plugin (which is already a requested feature e.g. https://github.com/1Password/shell-plugins/pull/141), but an older version of the CLI, if they were not to update to the latest.

In that case, there would be no actionable error message. I am wondering if we should be artificially triggering an error message by increasing the protocol version? Then they would get a "1Password CLI is outdated" error message when trying to use a locally built plugin against the CLI.

Reply:
This, however, is a secondary concern, and a situation that I expect to be rather rare. I'm fine with leaving this for later or even address it only if it becomes a problem, but let's track it with a ticket internally 😄

Reply:
If we're concerned about version mismatches then we could also upgrade the protocol version, which will result in a nicer error message.

Reply:
opened https://github.com/1Password/shell-plugins/issues/236 to track the protocol upgrade separately